greek/offer_marshmallow_a_cigar/intimacy/intimacy.md

```diff
@@ -0,0 +1,2 @@
+Το σκεφτεσαι περισσοτερο αναβωντας ακομα ένα τσιγαρο.
+Το ζαχαρωτο χερι εχει πνιγει στους καπνους αλλα πλεον δεν σε νοιαζει, εισαι οικειος μαζι του αφου εχουν περασει πεντε ολοκληρα λεπτα από τοτε που γνωριστηκατε. 
```

Comment:
@gooneraki, that was a step in the right direction! Just break it up into a few more lines, and you will be good to go.